### ISSUE TITLE
Keyboard focus is not visible on the control present below Open a terminal and clone the forked repository: A11y_Maow_Follow a workshop_Prerequisites_Focus Visible.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/moaw#118]
### GitHub Tags
#A11yTCS; #A11ySev2; #ChromiumEdge; #WCAG2.4.7; #Win11; #GH_Maow_Web_Mar24; #DesktopWeb; #Maow; #TabStop: #focus Visible; #A11yAuto; #A11yMAS;

### Environment Details:
Application Name: Maow
#URL: https://moaw.dev/workshop/create-workshop/?step=0#
OS: Windows 11 Enterprise 23H2 (OS build 22631.3296)
Microsoft Edge: Version 122.0.2365.92 (Official build) (64-bit)
### Repro Steps:
1. Hit the URL https://moaw.dev/workshop/create-workshop/
2. Create A Workshop screen will get open.
3. TAB till "Prerequisites" section.
4. Verify that the Keyboard focus is not visible on the control present below Open a terminal and clone the forked repository or not.

### Actual Result:
Keyboard focus is not visible on the control present below Open a terminal and clone the forked repository when we are navigating using the tab key.

### Expected Result:
The expected behavior is that keyboard focus should be visible when navigating through the tab key on the control present below Open a terminal and clone the forked repository

### User Impact:
People with motor disabilities, such as those who use a keyboard or switch device to navigate, are unable to access content due to the missing keyboard focus. This can lead to frustration and exclusion from the content.

### Attachment:
![Bug_118](https://github.com/microsoft/moaw/assets/124244478/c8dfee37-97d5-46dc-bb7e-b55b60922706)

![Screenshot (1682)](https://github.com/microsoft/moaw/assets/124244478/2d77ec86-9e04-40fa-9044-373304b2a934)

**Comment by sinedied** [comment]
@AnKushSingh05 this one has been fixed, could you validate it?

**Comment by AnKushSingh05** [comment]
Verified the issue in below environment
https://moaw.dev/workshop/create-workshop/?step=0#prerequisites
Issue is fixed. Hence closing the bug


https://github.com/microsoft/moaw/assets/124244478/f0163c0d-f522-45ef-9466-01fb9573bdeb
